plugins/rbac-backend-module-test/src/provider/TestProvider.ts

```diff
@@ -28,27 +29,28 @@ export class TestProvider implements RBACProvider {
   private connection?: RBACProviderConnection;
 
   static fromConfig(
-    config: Config,
-    options: {
+    deps: {
+      config: Config;
       logger: LoggerService;
-      schedule?: SchedulerServiceTaskRunner;
-      scheduler?: SchedulerService;
     },
+    options:
+      | { schedule: SchedulerServiceTaskRunner }
+      | { scheduler: SchedulerService },
   ): TestProvider {
-    const providerConfig = readProviderConfig(config);
+    const providerConfig = readProviderConfig(deps.config);
     let schedulerServiceTaskRunner;
 
-    if (options.scheduler && providerConfig.schedule) {
+    if ('scheduler' in options && providerConfig.schedule) {
       schedulerServiceTaskRunner = options.scheduler.createScheduledTaskRunner(
         providerConfig.schedule,
       );
-    } else if (options.schedule) {
+    } else if ('schedule' in options) {
       schedulerServiceTaskRunner = options.schedule;
     } else {
-      throw new Error('Neither schedule nor scheduler is provided.');
+      throw new InputError('Neither schedule nor scheduler is provided.');
```

Comment:
```suggestion
      throw new InputError(`No schedule provided via config for RBACTestProvider:${providerConfig.id}.`);
```

Reply:
Doesn't seem like there the `id` parameter exists in [TestProviderConfig](https://github.com/janus-idp/backstage-plugins/blob/705de117ab65e404e38d748b6c8c1a2a24d6be33/plugins/rbac-backend-module-test/src/provider/TestProvider.ts#L20-L24). 

I've changed the message to "No schedule provided via config for RBACTestProvider.". Lmk if that's okay!